api_v3/lib/types/EntryServerNode/live/KalturaLiveEntryServerNode.php

```diff
@@ -99,4 +106,22 @@ private function buildStreamInfoKeyValueArray($streamInfo = array())
 		
 		return $result;
 	}
+
+	public function toUpdatableObject($object_to_fill, $props_to_skip = array())
+	{
+		$dbRecordedProperties = $object_to_fill->getRecordedProperties();
+		$object_to_fill = parent::toUpdatableObject($object_to_fill, $props_to_skip);
+		if ($this->recordedProperties->duration > 0) {
+			/** @var LiveEntryServerNode $dbEntryServerNode */
+			$liveEntry = entryPeer::retrieveByPK($object_to_fill->getEntryId());
+			if (!$liveEntry)
```

Comment:
It's not possible to have live entry server mode and for the entry to be deleted no need to fetch for each update the live entry